python/ola/PidStore.py

```diff
@@ -1065,11 +1130,22 @@ def NameToValue(self, pid_name, esta_id=None):
     Returns:
       The value for this PID, or None if it wasn't found.
     """
-    pid = self.GetName(pid_name)
+    pid = self.GetName(pid_name, esta_id)
```

Comment:
We should probably add a test for this bug too?

Reply:
yeah, fair.   added

Reply:
Done.